apps/condo/domains/banking/schema/BankSyncTask.test.js

```diff
@@ -525,6 +525,33 @@ describe('BankSyncTask', () => {
             })
         })
 
+        it('expect error when creating BankAccount in 1c import', async () => {
```

Comment:
It might be better if you're done it something like that:
```suggestion
        it('throw an error if BankAccount already connected to requested property in file import', async () => {
```